test/faktory_worker/worker_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule FaktoryWorker.WorkerTest do
-  use ExUnit.Case, async: true
```

Comment:
We can no longer run these tests in async since we now have to use mox in global mode.